### ISSUE TITLE
OPSWAT Dead Link

### ISSUE THREAD

**Issue by thedeco** [inverse-inc/packetfence#5141]
**Describe the bug**
In the file:
docs/provisioner/opswat.asciidoc
It refers to two links that are no longer valid:
https://www.opswat.com/products/metadefender/endpoint/management/
https://gears.opswat.com/developers

I don't think "OPSWAT Metadefender Endpoint" is a product anymore or has been renamed.

**Comment by matsimon** [comment]
For future reference, unless the PacketFence is not maintained
OPSWAT seems to have made a bunch of renamings in over the time, but I seem to have found a working chain to based on a [blog post from 2016](https://www.opswat.com/blog/streamlined-metadefender-product-family-offers-unmatched-protection). Based on that it looks like Gears and Metadefender Endpoint references all point to "Metadefender Security Suite"

There is a link about [MetaDefender Endpoint](https://www.opswat.com/products/MetaDefender/endpoint) that redirects to what they offer now as [MetaAccess](https://www.opswat.com/products/metaaccess).

The documented service looks like a "as a service" product, hence why I guess that it is about the 2022 is rather "MetaAccess Cloud and SDP" product, not "MetaAcces NAC".

There is documentation here for MetaAccess: https://docs.opswat.com/

The question is, if the support for OPSWAT integration was kept up-to-date, or if it is defunct by now?

**Comment by nqb** [comment]
Hello,

We removed OPSWAT MetaDefender support: ccf84bb351eb6335929ee8fbbda1ce9032b36bff

**Comment by matsimon** [comment]
Thank you @nqb I just skimmed through old issues and that wasn't immediately visible.

As as there are still remainders in the Installation guide: If you don't mind, I'll prepare a pull request removing these references, should be an easy first PR on PacketFence for me. ;-)

Edit: After re-reading the linked pull request I realized that specifically "OPSWAT Metadefender Cloud Scanner" was removed, the rest remained and even the original author in the related PR #3852 mentions that the provisioner itself remains. 

However with all the renamings in OPSWAT and the little activity in Packetfence for that integration it could be a question to the PacketFence maintainers if they want to keep OPSWAT integration around. But that's not something an outside random contributor should decide upon. However I'm happy to help dropping old things. 😏
